moviepy/video/VideoClip.py

```diff
@@ -1245,8 +1245,8 @@ def __init__(
             subprocess_call(cmd, logger=None)
         except (IOError, OSError) as err:
             error = (
-                f"MoviePy Error: creation of {filename} failed because of the"
-                "following error:\n\n{err}.\n\n."
+                "MoviePy Error: creation of {filename} failed because of the"
+                f"following error:\n\n{err}.\n\n."
```

Comment:
I only discovered this accidentally by looking at my IDE word colouring, but definitely something for both of us to be aware of in the future:
The `f` before the string must be on the same line as the `{}` that you have the variable in because the way it works is through implicit concatenation of strings, so only the first line was an f-string. I've fixed it here and in the other place (line 77 of config.py) it occurred.